### PR TITLE
Adding sonata block context into the doc

### PR DESCRIPTION
**Pull request by eerison**
## Remove old sonata block doc

I am targeting this branch, because it's wrong in 4.x branch.

### CODE REVIEW

**Comment by eerison** [comment]
The doc is missing the example like

```
sonata_block:
    default_contexts: [sonata_page_bundle]
```

**Comment by eerison** [comment]
Screenshot 2022-09-05 at 15 34 54

File: docs/reference/installation.rst [hunk 1]
```diff
@@ -126,7 +126,7 @@ SonataAdminBundle Configuration
 
 .. code-block:: yaml
 
-    # config/packages/sonata_admin.yaml
+    # config/packages/sonata_page.yaml
```
Comment (jordisala1991) [inline docs/reference/installation.rst]: this line was correct.

File: docs/reference/installation.rst [hunk 2]
```diff
@@ -138,7 +138,12 @@ SonataAdminBundle Configuration
 SonataBlockBundle Configuration
 -------------------------------
 
-You need to use the context ``sonata_page_bundle`` in the SonataBlockBundle to add block into a Page.
+.. code-block:: yaml
+
+    # config/packages/sonata_page.yaml
```
Comment (jordisala1991) [inline docs/reference/installation.rst]:
```suggestion
    # config/packages/sonata_block.yaml
```

File: docs/reference/installation.rst (hunk 1 above)
Comment (eerison) [reply docs/reference/installation.rst]: Done!

File: docs/reference/installation.rst (hunk 2 above)
Comment (eerison) [reply docs/reference/installation.rst]: Done!

**Review by jordisala1991 — APPROVED** [review]

**Pull request merged**